src/workerd/io/trace-stream.c++

```diff
@@ -202,9 +202,9 @@ jsg::JsValue ToJs(jsg::Lock& js, const FetchEventInfo& info, StringCache& cache)
 }
 
 jsg::JsValue ToJs(jsg::Lock& js, const JsRpcEventInfo& info, StringCache& cache) {
-  static kj::StringPtr keys[] = {TYPE_STR};
+  static const kj::StringPtr keys[] = {TYPE_STR};
```

Comment:
Note: eventually we may want to convert these to using `JSG_STRUCT` now that the perf enhancement is there using `v8::DictionaryTemplate`. It'll actually be faster now to use that than to build these up manually this way. Doesn't need to be done here, of course, just mentioning for a future iteration.